.github/workflows/inshpect.yml

```diff
@@ -0,0 +1,42 @@
+name: Inshepct
+
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  doc:
+    runs-on: ubuntu-22.04
+
+    steps:
+      - name: Install tools
+        run: |
+          sudo apt update
+          sudo apt install -y fd-find
+          curl -sL https://github.com/TomWright/dasel/releases/download/v2.7.0/dasel_linux_amd64 -o /tmp/dasel
```

Comment:
Change this to use the same format as for `inshpect` and `ripgrep`:
- Full flags (`--output` vs `-o` etc.)
- Checksum